catroid/src/org/catrobat/catroid/formulaeditor/Formula.java

```diff
@@ -272,8 +272,6 @@ public String getResultForComputeDialog(Context context) {
 			} catch (InterpretationException interpretationException) {
 				return "ERROR";
 			}
-			interpretationResult *= 100;
-			interpretationResult = (double) (Math.round(interpretationResult) / 100f);
```

Comment:
Removing these lines changes the default compute output behavior. Such a change has to be discussed with the UX team first.

Current output: The result rounded up to two decimal places*
Your output: The result :smirk_cat:

\* It currently doesn't because of floating-point inaccuracy and the next line...